packages/db/src/index.ts

```diff
@@ -84,7 +84,6 @@ export {
   translatePostgresProviderToApiProvider,
 } from './translate/providers';
 export {
-  translateApiRuleToPostgresRule,
```

Comment:
This looks like a mistake?

Reply:
For that matter, *how* does this pass CI?      

.... it looks like most refs were changed to translateApiRuleToPostgresRuleRaw, but api/bin/serveUtils (which may not be tested) uses it.

Reply:
@npauzenga  I think this needs addressed.

Reply:
Hmm I think you're right. The difference between the two looks like `translateApiRuleToPostgresRuleRaw` does not remove nil properties where the other does. There could be implications in serve.js if we switch. Not sure and you're correct, it looks like serveUtils and serve.js aren't tested.

I'm tempted to just continue exporting `translateApiRuleToPostgresRule` here but it might be worth investigating why this change was made.

Reply:
So I think that's the right thing to do. It's the easiest but it also makes sense to me given then way `translateApiRuleToPostgresRule` has been refactored. It calls `translateApiRuleToPostgresRuleRaw` and removes nil properties. It seems like the intention was to make both available.

Given that the only use we're seeing for `translateApiRuleToPostgresRule` is in setting up a local instance of our API, I think it could go either way. Maintaining the original behavior seems fine to me but I'm open to pushback. 

https://github.com/nasa/cumulus/pull/2911/commits/fbbc1e475bf02f286ce9ae128788c08826f2909b

Reply:
Works for me.  I think if we want to address a refactor we should do it outside of this merge PR.